library/build.gradle

```diff
@@ -20,12 +20,11 @@ apply plugin: 'com.github.dcendents.android-maven'
 group = 'com.github.franmontiel'
 
 android {
-    compileSdkVersion 25
-    buildToolsVersion "26.0.2"
+    compileSdkVersion 27
 
     defaultConfig {
         minSdkVersion 15
-        targetSdkVersion 25
+        targetSdkVersion 27
```

Comment:
Can you target 28 without using support v28? Might be the way to go

Reply:
Nah, Lint gives an error if you try to do this (support library version should match the target SDK)